plugins/LilyPad/LilyPad.cpp

```diff
@@ -41,8 +41,8 @@
 #define FORCE_UPDATE_WPARAM ((WPARAM)0x74328943)
 #define FORCE_UPDATE_LPARAM ((LPARAM)0x89437437)
 
-// LilyPad version.
-#define VERSION ((0 << 8) | 11 | (0 << 24))
+// LilyPad version. Last updated on: November 29th, 2016
```

Comment:
> Last updated on: November 29th, 2016
Maybe this part isn't needed? It would be a PITA for future contributors if they want to keep updating the comment with the date, also timezone problems/commit merge date conflicts?